modules/oe_content_entity/modules/oe_content_entity_contact/config/install/field.field.oe_contact.oe_press.oe_address.yml

```diff
@@ -26,6 +26,8 @@ settings:
       override: hidden
     familyName:
       override: hidden
+    organization:
+      override: hidden
```

Comment:
Maybe we move these on a separate issue? We need to write an update path to o.Feel free to open a follow up.